addons/bestja_frontend_fpbz/static/src/css/custom.css

```diff
@@ -0,0 +1,389 @@
+}
+
+ul.nav-map li a .nav-map-point-label{
+    text-align: center;
+    display: block;
+    line-height: 1;
+}
+
+ul.nav-map li a.suwalki .nav-map-point-label,ul.nav-map li a.bialystok .nav-map-point-label{
+    margin-right: 3px;
+}
+
+ul.nav-map li a.suwalki .nav-map-point-label,ul.nav-map li a.chojnice 
+.nav-map-point-label,ul.nav-map li a.bialystok .nav-map-point-label, 
+ul.nav-map li a.radom .nav-map-point-label,ul.nav-map li a.nowe-bielice .nav-map-point-label{
+    float: left;
+}
+
+ul.nav-map li a.suwalki .nav-map-point-dot,ul.nav-map li a.chojnice 
+.nav-map-point-dot,ul.nav-map li a.bialystok .nav-map-point-dot,
+ul.nav-map li a.radom .nav-map-point-dot,ul.nav-map li a.nowe-bielice .nav-map-point-dot{
+    float: right;
+}
+
+ul.nav-map li a.nowe-bielice .nav-map-point-label{
+    width: 40px;
+}
+
+ul.nav-map li a.gorzow-wlkp .nav-map-point-label{
+    width: 40px;
+}
+
+ul.nav-map li a.ruda-slaska .nav-map-point-label{
+    width: 40px;
+}
+
+ul.nav-map li a.kielce .nav-map-point-label{
+    display: inline;
+}
+
+ul.nav-map li a:hover{
+    color: #f08d00;
+}
+
+ul.nav-map li a:hover .nav-map-point{
+    background: #f08d00;
+}
+
+ul.nav-map li a:hover .nav-map-point.nav-map-point-triangle{
+    background: 0;
+    border-bottom: 7px solid #f08d00;
+}
+
+.map_of_poland{
+    background: url('/bestja_frontend_fpbz/static/src/img/content/mapa_2.png') no-repeat;
+    width: 440px;
+    height: 402px;
+    margin: 0 auto!important;
+    padding: 0;
+    font-size: 12px;
+    font-weight: bold;
+    line-height: 1;
+    z-index: 2;
+    margin-bottom: 20px;
+}
+
+.text-right-only-desktop{
+    text-align: right;
+}
+
+.text-left-only-desktop{
+    text-align: left;
+}
+
+@media screen and (max-width: 991px) {
+    .text-right-only-desktop, .text-left-only-desktop{
+        text-align: center;
+    }
+}
+
+/* End of snippet: Map with cities with places where we can help */
+
+/* Snippet: a big quote */
+.quote_div blockquote p{
+    font-size: 28px;
+    font-style: italic;
+}
+
+.big_apostrophe{
+    font-family: 'Arial Bold', Arial; 
+    font-weight: 700; 
+    font-style: normal; 
+    font-size: 48px;
+}
+/* End of snippet: a big quote */
+
+/* Snippet: volunteers newsletter */
+.volunteers_newsletter{
+    
+}
+/* End of snippet: volunteers newsletter */
+
+
+/* Snippet: volunteeres pictures */
+.meet_our_volunteers button{
+    min-width: 290px;
+    background-color: #EC8D17;
+    color: #FFFFFF;
+    font-size: 20px;
+}
+/* End of snippet: volunteeres pictures */
+
+/* Footer */
+#footer_icons li{
+    list-style: none;
+    margin-left: 30px;
+    text-align: left;
+}
+
+#footer_icons li a.facebook{
+    background-image: url('/bestja_frontend_fpbz/static/src/img/content/stopka_ikonki.png');
+    display: inline-block;
+    background-position: 0 0;
+    background-repeat: no-repeat;
+    width: 35px;
+    height: 45px;
+    vertical-align: middle;
+    margin-left: 7px;
+}
+
+#footer_icons li a{
+    
```

Comment:
Shouldn’t some properties (like height) be put here instead of the per-icon class?